libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -612,21 +612,69 @@ $button-normal-active-shadow: transparentize($button-normal-active-border-color,
     display: flex;
     flex: 1 0 auto;
 
+    @mixin icon-state {
+        &.disabled {
+            color: $gd-color-disabled;
+        }
+
+        &.is-normal {
+            color: $gd-color-link;
+        }
+    }
+
+    @mixin hover-effect {
+        &:hover {
+            z-index: 100;
+            box-shadow: 0 0 0 1px $gd-color-highlight;
+
+            @include icon-state;
+        }
+    }
+
     .icon-sources {
         border-bottom-right-radius: 0;
         border-top-right-radius: 0;
+        margin-top: 2px;
+        margin-left: 2px;
```

Comment:
don't we need to set `margin-top` & `margin-left` to `.icon-model` and `.icon-cloud-upload` as well?

Reply:
we dont need to set it the icon of model is smaller than sources and make it miss-align , icon-cloud-upload is aligned already so I just fix like a comment for model icon